src/animationBuilder.tsx

```diff
@@ -25,12 +26,52 @@ const mockTargetValues: LayoutAnimationsValues = {
   currentBorderRadius: 0,
 };
 
+function maybeReportOverwrittenProperties(
+  layoutAnimationStyle: StyleProps,
+  style: NestedArray<StyleProps>,
+  displayName: string
+) {
+  function getCommonProperties(
```

Comment:
Does function `getCommonProperties` need to be nested inside `maybeReportOverwrittenProperties`?

Reply:
No, it doesn't